### PR TITLE
Addition of issuer to the assert response

### PR DESCRIPTION
**Pull request by PiotrJachowicz**

### CODE REVIEW

File: test/saml2.coffee [hunk 1]
```diff
@@ -379,6 +379,8 @@ describe 'saml2', ->
 
       sp.post_assert idp, request_options, (err, response) ->
         assert not err?, "Got error: #{err}"
+        console.log 'hello'
```
Comment (nkumarcc) [inline test/saml2.coffee]: Can you get rid of this in PR?

File: lib/saml2.coffee [hunk 2]
```diff
@@ -477,6 +477,10 @@ parse_authn_response = (saml_response, sp_private_keys, idp_certificates, allow_
         assertion_attributes = parse_assertion_attributes decrypted_assertion
         user = _.extend user, pretty_assertion_attributes(assertion_attributes)
         user = _.extend user, attributes: assertion_attributes
+
+        issuer = dom.getElementsByTagNameNS(XMLNS.SAML, 'Issuer')
+        request.issuer = issuer[0].firstChild?.data if issuer.length is 1
```
Comment (nkumarcc) [inline lib/saml2.coffee]: Did `request.issuer` field already exist, it just wasn't set? I know you said you had seen it before

**Review by nkumarcc — APPROVED** [review]
Tbh this looks great to me! It also looks like this should be perfectly secure, as it nested underneath a guarantee we have a decrypted assertion.

Let's do it! Asked some questions in the comments + asked for deleting some commented code you had. After doing so can you rebuild the JS and merge this yourself?

**Comment by nkumarcc** [comment]
Notable, for you to play with this locally I think you'll have to rerun `npm install` locally, you should rebuild `api` container

File: lib/saml2.coffee (hunk 2 above)
Comment (PiotrJachowicz) [reply lib/saml2.coffee]: This was actually my bad, this should be passed in in the object a few lines below. Will be fixed in a commit in a moment

File: test/saml2.coffee (hunk 1 above)
Comment (PiotrJachowicz) [reply test/saml2.coffee]: Done

**Pull request merged**